.github/workflows/scripts/native-image-tests.sh

```diff
@@ -10,14 +9,30 @@ set -eo pipefail
 # Get git repo root
 scriptDir=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
 cd "${scriptDir}/../../.."
+IFS='-' read -ra MATRIX_SUBSTRINGS <<< "$MODULE_UNDER_TEST"
```

Comment:
Leaving IFS set to the non-default value can cause issues in the rest of the script.

Consider:
```
OLD_IFS=IFS
IFS=...
IFS=OLD_IFS
```

Reply:
Done, thank you!